gcc/rust/ast/rust-ast-dump.cc

```diff
@@ -746,7 +745,24 @@ Dump::visit (Function &function)
 
 void
 Dump::visit (TypeAlias &type_alias)
-{}
+{
+  // TODO does this represent associated type
```

Comment:
Good catch, luckily with our AST class hierarchy for associated types we have: AST::TraitItemType

Reply:
I just follow the syntax description in the Rust reference. It seems quite useful to get all the cases.